server/backends/github/github.go

```diff
@@ -37,7 +37,7 @@ var (
 
 	clientID     = flag.String("github.client_id", "", "The client ID of your GitHub Oauth App. ** Enterprise only **")
 	clientSecret = flag.String("github.client_secret", "", "The client secret of your GitHub Oauth App. ** Enterprise only **", flag.Secret)
-	accessToken  = flag.String("github.access_token", "", "The GitHub access token used to post GitHub commit statuses. ** Enterprise only **", flag.Secret)
+	accessToken  = flag.String("github.access_token", "", "A hard-coded GitHub access token to post GitHub commit statuses. Enterprise customers should use an GitHub app installation instead.", flag.Secret)
```

Comment:
Would avoid mentioning the GH app here since we only officially support it on cloud atm, also hard-coded usually refers to things like `const` that are compiled into the code and can't be changed externally iiuc.

```suggestion
	accessToken  = flag.String("github.access_token", "", "The GitHub access token used to post GitHub commit statuses. This is intended as a convenience option and can be set to a personal access token (PAT) for single-user deployments or a shared machine account PAT for multi-user deployments.", flag.Secret)
```